README.md

```diff
@@ -1,67 +1,68 @@
-# Basic CRUD App with Node + React
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-This example app shows how to create a Node.js API and display its data with a React UI. To follow along step-by-step, [check out the blog post](https://developer.okta.com/blog/2018/07/10/build-a-basic-crud-app-with-node-and-react).
+## Available Scripts
 
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
@jon-rtr I think this is really the only line of the README that needs to change. Could you please restore the rest of the readme?